### PR TITLE
errorprone :: CatchingUnchecked

### PR DESCRIPTION
**Pull request by dev-mlb**

### CODE REVIEW

**Review by sambish5 — DISMISSED** [review]

File: src/main/java/emissary/place/MultiFileUnixCommandPlace.java [hunk 1]
```diff
@@ -605,6 +605,7 @@ protected List<IBaseDataObject> processData(IBaseDataObject tData) throws Resour
      * @param len length of data to use
      * @return attachments
      */
+    @SuppressWarnings("CatchingUnchecked")
```
Comment (DonResnik) [inline src/main/java/emissary/place/MultiFileUnixCommandPlace.java:608]: why does this method still catch Exception and not RuntimeException?

File: src/main/java/emissary/pool/PayloadLauncher.java [hunk 2]
```diff
@@ -21,6 +21,7 @@ public class PayloadLauncher {
     private static final Logger logger = LoggerFactory.getLogger(PayloadLauncher.class);
 
 
+    @SuppressWarnings("CatchingUnchecked")
```
Comment (DonResnik) [inline src/main/java/emissary/pool/PayloadLauncher.java:24]: why does this method catch Exception and not RuntimeException?

Comment (dev-mlb) [reply src/main/java/emissary/pool/PayloadLauncher.java:24]: `pool.borrowAgent()` throws Exception

**Review by sambish5 — APPROVED** [review]
changes lgtm

**Pull request merged**